wrapper.js

```diff
@@ -134,6 +153,14 @@ function setupMethods (soljson) {
       throw e;
     }
     removeFunction(cb);
+    if (isVersion6) {
```

Comment:
Actually could run `reset` in every case, as it should not have an adverse effect on pre-0.6.0.

Reply:
Yea, just need to create it also for `!isVersion6`

Reply:
Yeah forgot that it wasn't exposed :wink: